### PR TITLE
Update jubeatfesto.html

### PR DESCRIPTION
**Pull request by ikaros14**
Finally today ``Ave.`` comes.

### CODE REVIEW

**Comment by mon** [comment]
Sorry, by merging the final hex edits this now has conflicts.... Also several typos (Tutorial -> Turorial)

I can fix everything in ~6-10 hours, but if you fix them earlier I can merge from my phone.

**Comment by ikaros14** [comment]
oh my, I had no idea until now 🤣
corrected typos and included a new patch for Subject38

File: jubeatfesto.html [hunk 1]
```diff
@@ -8,28 +8,82 @@
         <script type="text/javascript">
             window.addEventListener("load", function() {
                 new PatchContainer([
-                    new Patcher("jubeat.dll", "2022-05-24", [
+                    new Patcher("jubeat.dll", "2022-05-24, 2022-02-16, 2021-10-19", [
                         {
-                            name: "Skip Tutorial",
-                            patches: [{ offset: 0xD746F, off: [0x0F, 0x85], on: [0x90, 0xE9] }],
+                          name: "Disable Tutorial",
+                          patches: [{offset: 0xD746F, off: [0x0F, 0x85, 0x7F, 0x01, 0x00, 0x00], on: [0xE9, 0x80, 0x01, 0x00, 0x00, 0x90]}]
```
Comment (mon) [inline jubeatfesto.html:14]: I notice this hex edit is actually different, do you know if it works any differently? everything else is OK

Comment (ikaros14) [reply jubeatfesto.html:14]: it works same

**Pull request merged**